content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -1790,6 +1790,22 @@ uses this annotation.
 See [annotations](https://kubernetes-sigs.github.io/aws-load-balancer-controller/latest/guide/service/annotations/)
 in the AWS load balancer controller documentation.
 
+### service.beta.kubernetes.io/load-balancer-security-group {#service-beta-kubernetes-io-aws-load-balancer-security-groups}
+
+Example: `service.beta.kubernetes.io/aws-load-balancer-security-groups: "sg-53fae93f, sg-8725gr62r"`
+
+Used On: Service
```

Comment:
```suggestion
Used on: Service
```
For consistency in context

Reply:
Noted. I'll update the change. Any other change you would like to suggest @windsonsea ?

Reply:
Need to wait for approvers come in. This can be marked with `Resolve conversation`.